packages/client/src/index.ts

```diff
@@ -20,7 +20,7 @@ export class MonacoLanguageClient extends BaseLanguageClient {
     protected readonly connectionProvider: IConnectionProvider;
 
     constructor({ id, name, clientOptions, connectionProvider }: MonacoLanguageClientOptions) {
-        super(id || name.toLowerCase(), name, clientOptions);
+        super((id ?? 'unknown-id') || name.toLowerCase(), name, clientOptions);
```

Comment:
I'm confused by the double usage of `??` and `||`

Reply:
My mistake. What confuses me now is that eslint finds problems I don't see locally 😐

Reply:
Done

Reply:
It seems to be still failing :thinking: